compute/instances/reference-content/understanding-instance-pricing.mdx

```diff
@@ -0,0 +1,97 @@
+
+You can power off your Instance or delete it at any time: no minimum commitment or notice period is required.
+
+At the end of each calendar month you will be invoiced for your hourly billed usage that month.
+
+## Flexible IP addresses and storage volumes
+
+### Flexible IP address billing 
+
+When you create an Instance, by default a [flexible IP](/compute/instances/concepts/#flexible-ip/) is created for it at the same time. This gives the Instance a public IP address. A flexible IP address is billed separately to the Instance itself, and has its own hourly rate. You can see the hourly rate for a flexible IP address on the Instance [creation page](https://console.scaleway.com/instance/servers/create) of the Scaleway console, or on the dedicated [pricing page](https://www.scaleway.com/en/pricing/?tags=available) of our website.
+
+Your hourly billing for a flexible IP address begins when you reserve the IP address (either during the creation of an Instance, or by creating a [new](/compute/instances/how-to/use-flexips/#how-to-create-a-new-flexible-ip-address) flexible IP address) and ends when you [delete](/compute/instances/how-to/use-flexips/#how-to-delete-a-flexible-ip-address) the address from your account.
+
+<Message type="note">
+  - You can choose to create an Instance without a flexible IP address via the **Advanced settings** option.
+  - Billing of an Instance stops when you power it off, but the Instance's flexible IP remains reserved to your account nonetheless. You will continue to be billed for the flexible IP even if your Instance is powered off, unless you delete the flexible IP.
+  - When you delete an Instance via the console, you are asked whether you want to also delete its flexible IP. If you delete it, billing for this IP will end. If you keep it in your account, billing continues.
+</Message>
+
+You can detach a flexible IP address from your Instance and delete it at any time: no minimum commitment or notice period is required.
+
+At the end of each calendar month, you will be invoiced for your hourly billed flexible IP usage.
+
+### Storage volume billing 
+
+When you create an Instance, you also need to create a certain amount of storage for it, in the form of [local or block volumes](/compute/instances/concepts/#volumes). Volumes are billed separately to Instances, and have their own hourly rates. You can see the hourly rate for different types of volume on the Instance [creation page](https://console.scaleway.com/instance/servers/create) of the Scaleway console, or on the dedicated [pricing page](https://www.scaleway.com/en/pricing/?tags=available) of our website.
+
+Your hourly billing for a storage volume begins when you create the volume (either during the creation of an Instance, or by creating a [new block volume](/storage/block/how-to/create-a-volume/)), and ends when you [delete](/storage/block/how-to/delete-a-volume/) the volume from your account.
```

Comment:
```suggestion
Your hourly billing for a storage volume begins when you create the volume (either during the creation of an Instance, or by creating a [new block volume](/storage/block/how-to/create-a-volume/), and ends when you [delete](/storage/block/how-to/delete-a-volume/) the volume from your account.
```

Reply:
this one is meant to be there to close the brackets in the sentence